UI/TechTreeWnd.cpp

```diff
@@ -58,6 +58,20 @@ namespace {
                default_pts * 12,        StepValidator<int>(1));
         db.Add("UI.research.listbox.column-widths.description", UserStringNop("OPTIONS_DB_UI_TECH_LISTBOX_COL_WIDTH_DESCRIPTION"),
                default_pts * 18,        StepValidator<int>(1));
+
+        // Default status for TechTreeControl filter.
+        db.Add<bool>("UI.research.tech-tree-control.status.unresearchable",
+                     UserStringNop("OPTIONS_DB_UI_TECH_TREE_STATUS_UNRESEARCHABLE"),
+                     false,         Validator<bool>());
```

Comment:
Not sure is tabulating is desired (seems there is contention here), this spacing is off from following portions.